IdentityCore/src/util/NSError+MSIDExtensions.m

```diff
@@ -76,4 +76,17 @@ - (nullable NSString *)msidSubError
     return self.userInfo[self.msidErrorConverter.subErrorKey];
 }
 
+- (nullable NSDictionary*)getStatusCodeUserInfoFromUnderlyingErrorCode
```

Comment:
Would be good to add some unit tests for this one as well

Reply:
Added new unit tests